LifeCycleAssessment_oM/Results/MaterialResult.cs

```diff
@@ -0,0 +1,77 @@
+﻿using BH.oM.Analytical.Results;
+using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BH.oM.LifeCycleAssessment.Results
+{
+    [Description("Life Cycle Assessment result for a Material/Environmental Product Declaration evaluated.")]
+    public class MaterialResult : IResult, IImmutable
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Name of the physical material evaluated.")]
+        public virtual string MaterialName { get; }
+
+        [Description("Name of the Environmental Product Declaration evaluated.")]
+        public virtual string EPDName { get; }
```

Comment:
Avoiding acronyms and matching `EnvironmentalProductDeclercationField`, this should be:
```suggestion
        public virtual string EnvironmentalProductDeclarationName { get; }
```